web/src/app/loading/components/LoadingButton.tsx

```diff
@@ -34,7 +34,7 @@ const LoadingButton = (props: LoadingButtonProps): JSX.Element => {
         disabled={loading || disabled}
         type={noSubmit ? 'button' : 'submit'}
       >
-        {buttonText || (!attemptCount ? 'Confirm' : 'Retry')}
+        {!attemptCount ? buttonText || 'Confirm' : 'Retry'}
```

Comment:
This is not preferable moving forward as we want the button to say a specific text ALWAYS, and not have it change to "Retry" on the first click.

Reply:
This is handled by the `alert` prop on FormDialog -- once we have a token we set that prop to `true` which forces the button to always say `DONE`

Reply:
Changing this logic causes playwright tests to fail as other parts of the application count on `Retry` overriding the `buttonText` after the first attempt